src/Command/CheckScanCommand.php

```diff
@@ -185,19 +193,23 @@ public function execute(InputInterface $input, OutputInterface $output): int
                 );
 
                 $statusCode = $statusResponse->getStatusCode();
-                if ($statusCode === Response::HTTP_CREATED) {
+                if ($disableConditionalSkipScan === false && $statusCode === Response::HTTP_CREATED) {
                     break;
                 }
```

Comment:
Shall we return something here? It will look like no progress is made if they have disabled conditional skip scan

Reply:
@sweoggy @philomather

Reply:
Does the responses with Response::HTTP_CREATED still contain progress data in body?

And yes, should probably print something along the lines of "Scan queues are currently long, but you have disabled conditional scan, will wait indefinitely until scan is complete"

Reply:
No it does not, we could add that actually. Then we can be certain it always exists. 

Cool will add that then :+1:

Reply:
Fixed 69d4c5a9e8d9f93ff61c301e7bf78d7330a75d2d